NEWS.md

```diff
@@ -1,3 +1,55 @@
+# New in snapd 2.67
+* AppArmor prompting (experimental): allow overlapping rules
+* Registry view (experimental): Changes to registry data (from both users and snaps) can be validated and saved by custodian snaps
+* Registry view (experimental): Support 'snapctl get --pristine' to read the registry data excluding staged transaction changes
+* Registry view (experimental): Put registry commands behind experimental feature flag
+* Components: Make modules shipped/created by kernel-modules components available right after reboot
+* Components: Add tab completion for local component files
+* Components: Allow installing snaps and components from local files jointly on the CLI
+* Components: Allow 'snapctl model' command for gadget and kernel snaps
+* Components: Add 'snap components' command
+* Components: Bug fixes
+* eMMC gadget updates (WIP): add syntax support in gadget.yaml for eMMC schema
+* Support for ephemeral recovery mode on hybrid systems
+* Support for dm-verity options in snap-bootstrap
+* Support for overlayfs options and allow empty what argument for tmpfs
+* Enable ubuntu-image to determine the size of the disk image to create
+* Expose 'snap debug' commands 'validate-seed' and 'seeding'
+* Add debug API option to use dedicated snap socket /run/snapd-snap.socket 
+* Hide experimental features that are no longer required (accepted/rejected)
+* Mount ubuntu-save partition with no{exec,dev,suid} at install, run and factory-reset
+* Improve memory controller support with cgroup v2
+* Support ssh socket activation configurations (used by ubuntu 22.10+)
+* Fix generation of AppArmor profile with incorrect revision during multi snap refresh 
```

Comment:
The issue could also happen if the refreshes were in separate changes so maybe this is more accurate (unless it's too long?)
```suggestion
* Fix generation of AppArmor profile with incorrect revision during concurrent refreshes of snaps using content interface
```

Reply:
Thanks, you version is better, at this point I am not going to re-trigger all testing for this change.